pytorch_lightning/trainer/connectors/logger_connector/logger_connector.py

```diff
@@ -270,7 +270,7 @@ def get_evaluate_epoch_results(self, test_mode):
         self.prepare_eval_loop_results()
 
         # log results of test
-        if test_mode and self.trainer.is_global_zero and self.trainer.verbose_test:
+        if self.trainer.evaluating and self.trainer.is_global_zero and self.trainer.verbose_evaluate:
```

Comment:
looks like `test_mode` is not used now. Can you remove it from the function arguments and update wherever it's called??

Reply:
Indeed, thank you. Addressed in the latest commit.